packages/orbit-components/src/TextLink/TextLink.stories.tsx

```diff
@@ -1,292 +1,219 @@
-    info: "It inherits the visual style of the parent paragraph. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "An example of usage of TextLink in Text component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onClick", "external", "asComponent", "stopPropagation"],
+    },
   },
-};
 
-TextLinkInText.args = {
-  type: TEXT_TYPES.CRITICAL,
-};
+  args: {
+    type: TYPE_OPTIONS.CRITICAL,
+  },
 
-TextLinkInText.argTypes = {
-  type: {
-    options: Object.values(TEXT_TYPES),
-    control: {
-      type: "select",
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const Accessibility = ({ children, title }) => {
-  return (
-    <TextLink title={title} onClick={action("clicked")} type="primary">
-      {children}
-    </TextLink>
-  );
-};
+export const Playground: Story = {
+  render: ({ rel, type, children, iconRight, iconLeft, ...args }) => {
+    const IconRight = typeof iconRight === "string" && getIcon(iconRight);
+    const IconLeft = typeof iconLeft === "string" && getIcon(iconLeft);
+
+    return (
+      <Box padding="300" background={type === TYPE_OPTIONS.WHITE ? "inkLight" : "cloudLight"}>
+        <TextLink
+          {...args}
+          type={type}
+          rel={validate(rel!)}
+          iconRight={IconRight && <IconRight />}
+          iconLeft={IconLeft && <IconLeft />}
+        >
+          {children}
+        </TextLink>
+      </Box>
+    );
+  },
 
-Accessibility.story = {
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+  args: {
+    ...LinkWithLeftIcon.args,
+    ...LinkWithRightIcon.args,
+    type: TYPE_OPTIONS.SECONDARY,
+    size: SIZE_OPTIONS.SMALL,
+    external: true,
+    rel: "",
+    iconRight: "ChevronForward",
+    iconLeft: "",
+    tabIndex: "",
+    stopPropagation: false,
+    standAlone: false,
+    noUnderline: false,
+    title: "link title",
+    ariaCurrent: "text-link",
+    asComponent: "a",
+    id: "link-id",
   },
-};
 
-Accessibility.args = {
-  children: "Primary link",
-  title: "Clarify purpose of a link for screen readers",
+  argTypes: {
+    ...LinkWithLeftIcon.argTypes,
+    ...LinkWithRightIcon.argTypes,
+    ...TextLinkInText.argTypes,
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
+    size: {
+      options: Object.values(SIZE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
+  },
 };
 
-export const Rtl = () => (
-  <RenderInRtl>
-    <TextLink onClick={action("clicked")} href="#" iconRight={<Icons.ChevronForward />}>
-      Link
-    </TextLink>
-  </RenderInRtl>
-);
-
-Rtl.story = {
-  name: "RTL",
+export const Rtl: Story = {
+  render: args => (
+    <RenderInRtl>
+      <TextLink {...args} iconRight={<Icons.ChevronForward />}>
```

Comment:
It would look visually "better", but I think we want to point to a reversed order of components as much as possible in RTL. Using `ChevronBackward` it would only look like aligned to the right. 🤔